generators/entity/logic/enum.mjs

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Entity generator should be dumb, and is used only for adding/editing entities using prompts.
This should be placed at `generators/base-application` or `lib`.

Reply:
not fully agree: logic owned by a generator (here, enum is closer to entity than anything else) should be as close as possible to the one that is the most suited.
What are the argument against it?

Reply:
moved there